### ISSUE TITLE
WCAG 2.1 1.4.3 -- Insufficient Color Contrast

### ISSUE THREAD

**Issue by KimPlausible** [swagger-api/swagger-ui#5247]
### Q&A (please complete the following information)
 - OS: Windows
 - Browser: Chrome 
 - Version: 73.0.3683.75
 - Method of installation: npm
 - Swagger-UI version: 3.17.6
 - Swagger/OpenAPI version: latest

### Describe the bug you're encountering

The color of the text and the color of the background are not in sufficient contrast to each other. This can give problems for users with visual impairments & color blindness, but also can affect a lot of other users, leading to important text potentially being overlooked.
### To reproduce...

Run Chrome Accessibility assessment or Siteimprove extension. See below screenshots for details.

### Expected behavior

The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:
- Large-scale text and images of text have a contrast ratio of at least 3:1
- Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement
- Text that is part of a logo or brand name has no minimum contrast requirement

### Screenshots

**Authorize** has insufficient color contrast of 2.03 (foreground color: #49cc90, background color: #ffffff, font size: 10.5pt, font weight: bold).
![image](https://user-images.githubusercontent.com/12721313/54622799-78a88200-4a27-11e9-9232-0647c5bb74e5.png)
**GET** has insufficient color contrast of 2.31 (foreground color: #ffffff, background color: #61affe, font size: 10.5pt, font weight: bold).
![image](https://user-images.githubusercontent.com/12721313/54622839-90800600-4a27-11e9-82ac-399b64d27e32.png)
**(query)** has insufficient color contrast of 3.65 (foreground color: #808080, background color: #eff7ff, font size: 9.0pt, font weight: bold).
![image](https://user-images.githubusercontent.com/12721313/54623231-60853280-4a28-11e9-9a50-7a95d81248ee.png)
**DELETE** has insufficient color contrast of 3.62 (foreground color: #ffffff, background color: #f93e3e, font size: 10.5pt, font weight: bold).
![image](https://user-images.githubusercontent.com/12721313/54623507-dd181100-4a28-11e9-9f94-3bb2e35ad89b.png)
**POST** has insufficient color contrast of 2.03 (foreground color: #ffffff, background color: #49cc90, font size: 10.5pt, font weight: bold).
![image](https://user-images.githubusercontent.com/12721313/54623598-15b7ea80-4a29-11e9-8854-3fe9fcc30091.png)
**Example Value** has insufficient color contrast of 4.21 (foreground color: #337ab7, background color: #eff7ff, font size: 9.0pt, font weight: bold).
![image](https://user-images.githubusercontent.com/12721313/54624027-d6d66480-4a29-11e9-8294-23cfdb89d722.png)

### Additional context or thoughts

Sufficient to provide control with sufficient contrast ratio that allows users to switch to a presentation that uses sufficient contrast or provide Style Switcher to switch to High Contrast.
- https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/SL13

**Comment by KimPlausible** [comment]
**Pri 2** (Not blocking, high impact): Authorize, GET, POST
**Pri 3** (Not blocking, low impact): (query), DELETE, Example Value

**Comment by mgifford** [comment]
This seems like such a trivial one to fix. We're seeing these basic colour contrast issues in our tests too. Please make sure that the colors conform to This is one of many good tools to test foreground/background colours and suggest better alternatives https://contrast-finder.tanaguru.com/
